astroquery/eso/core.py

```diff
@@ -591,6 +593,13 @@ def retrieve_data(self, datasets, continuation=False, destination=None):
         datasets_to_download = []
         files = []
 
+        calib_options = {'none': '', 'raw': 'CalSelectorRaw2Raw',
+                         'processed': 'CalSelectorRaw2Master'}
```

Comment:
There are two possibilities for calibrations, raw and processed (though I never managed to get master calib with the second option). So I added an option with three possible values.